packages/app/src/runner/selector-playground/SelectorPlayground.cy.tsx

```diff
@@ -110,42 +110,75 @@ describe('SelectorPlayground', () => {
     cy.get('@copy').click()
     cy.get('@copy').should('be.focused')
 
+    cy.get('[data-cy="playground-copy"]').trigger('mouseenter')
+    cy.get('[data-cy="selector-playground-tooltip"]').should('be.visible').contains('Copy to clipboard')
+    cy.percySnapshot('Copy to clipboard hover tooltip')
     cy.get('[data-cy="playground-copy"]').click()
-    cy.get('[data-cy="playground-copy-tooltip"]').should('be.visible').contains('Copied to clipboard')
+    cy.get('[data-cy="selector-playground-tooltip"]').should('be.visible').contains('Copied!')
+    cy.percySnapshot('Copy to clipboard click tooltip')
 
     cy.wrap(copyStub).should('have.been.calledWith', '.foo-bar')
   })
 
-  it('prints nothing to console when no selected elements found', () => {
-    mountSelectorPlayground()
+  it('prints elements when selected elements found', () => {
+    const { autIframe } = mountSelectorPlayground()
+
     cy.spy(logger, 'logFormatted')
+    cy.stub(autIframe, 'getElements').callsFake(() => Array(2))
+
     cy.get('[data-cy="playground-selector"]').clear().type('.foo-bar')
 
-    cy.get('[data-cy="playground-print"]').as('print')
-    cy.get('@print').click().then(() => {
+    cy.get('[data-cy="playground-print"]').trigger('mouseenter')
+    cy.get('[data-cy="selector-playground-tooltip"]').should('be.visible').contains('Print to console')
+    cy.percySnapshot('Print to console hover tooltip')
+
+    cy.get('[data-cy="playground-print"]').click()
+    cy.get('[data-cy="selector-playground-tooltip"]').should('be.visible').contains('Printed!')
+    cy.percySnapshot('Print to console click tooltip')
+
+    cy.then(() => {
       expect(logger.logFormatted).to.have.been.calledWith({
         Command: `cy.get('.foo-bar')`,
-        Yielded: 'Nothing',
+        Elements: 2,
+        Yielded: undefined, // stubbed dom does not actually return anything
       })
     })
-
-    cy.get('[data-cy="playground-print-tooltip"]').should('be.visible').contains('Printed to console')
   })
 
-  it('prints elements when selected elements found', () => {
-    const { autIframe } = mountSelectorPlayground()
-
+  it('prints nothing to console when no selected elements found', () => {
```

Comment:
This test seems to be checking the behavior for the invalid state, not the "no selected elements found" state - I'm surprised that `cy.type('.foo-bar')`, wonder if this is related to resetting validity. Or does this test have the wrong name?

Reply:
Zach and I talked offline a bit, the issue I'm seeing was not introduced in this PR, so I've created https://cypress-io.atlassian.net/browse/UNIFY-1727 to follow up.